### DISCUSSION TITLE
how can I handover github account details to the docker container

### DISCUSSION THREAD

**Discussion by elschman** [aquasecurity/trivy#6170] [Q&A]
### Question

I need to check a github repo with the trivy docker ...
The repo is locked for unknown, so I do have to handover the login credentials from the jenkins run.
How can I do this to scan the repo and not the already dirty workspace ?

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

**Comment by DmitriyLewen** [comment]
Hello @elschman 
Thanks for your interest to Trivy!

You can use `GITHUB_TOKEN` - https://aquasecurity.github.io/trivy/v0.49/docs/target/repository/#scanning-private-repositories

Regards, Dmitriy
